src/Chapter1.hs

```diff
@@ -430,14 +433,22 @@ Implement the function that takes an integer value and returns the next 'Int'.
 >>> next (-4)
 -3
 
+{-
+@
+next :: Int -> Int
+next x = (x + 1)
+@
+-}
+
 ♫ NOTE: The current function body is defined using a special function called
   "error". Don't panic, it is not broken. 'error' is like a placeholder, that
   evaluates to an exception if you try evaluating it. And it also magically fits
   every type ｡.☆.*｡. No need to worry much about "error" here, just replace the
   function body with the proper implementation.
 -}
+
 next :: Int -> Int
-next x = error "next: not implemented!"
+next x = x+1
```

Comment:
Nice! 👍🏼 
In Haskell functions and operators are always separated from arguments with spaces:
```suggestion
next x = x + 1
```